app/src/main/java/unipd/se18/ocrcamera/PreProcessing.java

```diff
@@ -0,0 +1,369 @@
+package com.example.imageprocessing;
+
+import android.graphics.Bitmap;
+import android.graphics.Matrix;
+import android.util.Log;
+import com.example.imageprocessing.exceptions.ConversionFailedException;
+
+import org.opencv.core.Core;
+import org.opencv.core.Mat;
+import org.opencv.core.MatOfInt4;
+import org.opencv.core.Scalar;
+import org.opencv.imgproc.Imgproc;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.opencv.core.CvType.CV_16S;
+import static org.opencv.core.CvType.CV_16U;
+import static org.opencv.core.CvType.CV_32F;
+import static org.opencv.core.CvType.CV_32S;
+import static org.opencv.core.CvType.CV_64F;
+import static org.opencv.core.CvType.CV_8S;
+import static org.opencv.core.CvType.CV_8U;
+import static org.opencv.core.CvType.CV_8UC1;
+
+/**
+ * Class used to process the image before passing the image to the OCR
+ */
+public class PreProcessing implements PreProcessingMethods {
+
+    //Tag used to identify the log
+    private final String TAG = "PreProcessing";
+
+
+    /**
+     * Constructor of the class which initialize the openCV library
+     * @author Thomas Porro (g1)
+     */
+    public PreProcessing() {
+        //TODO verify if the library is correctly loaded
```

Comment:
Why is it only a "to do" and it isn't code this check? Is it not important to do?